Makefile

```diff
@@ -290,6 +290,7 @@ ENVTEST ?= $(LOCALBIN)/setup-envtest
 ## Tool Versions
 KUSTOMIZE_VERSION ?= v5.2.1
 CONTROLLER_TOOLS_VERSION ?= v0.14.0
+ENVTEST_VERSION ?= release-0.18
```

Comment:
I think some CI tests were failing because of this so I changed this before the operator-sdk upgrade. But this should be superseded by https://github.com/openshift/vertical-pod-autoscaler-operator/pull/186 :)